### DISCUSSION TITLE
Interact with a deployed smart contract

### DISCUSSION THREAD

**Discussion by willywilly12345** [filecoin-project/community#615] [Q&A]
On what explorer can i interact with a deployed smart contract? I have tried zondax, it shows all the contract functions but it requests for a transaction fee even when i try to interact with te balanceOf function, which should be free correct? Filecoin does not have yet an Etherscan type of explorer where you can simply interact with the deployed smart contracts? Or maybe that the contract is not yet verified? Thank you fam

**Comment by longfeiWan9** [comment]
> I have tried zondax, it shows all the contract functions but it requests for a transaction fee even when i try to interact with te balanceOf function, which should be free correct?
I tried this feature on zondax, I can call read/view methods without paying a fee. But definitely, we can improve those interactions better, for example, decoding the returned result.
